src/TCPFooter.js

```diff
@@ -0,0 +1,33 @@
+
+import styles from './TCPFooter.module.css';
+
+export default function TCPFooter() {
+
```

Comment:
@seanno Do we want to add the "Powered by CommonHealth" with a link back to commonhealth.org? I know we have it up in the about info but I think JP wanted to call it out separately.

Reply:
Just spoke with JP and he is fine releasing it without the Powered by CommonHealth. We are going to see if Naomi can come up with a graphic we can use.